gradle/project.go

```diff
@@ -0,0 +1,84 @@
+package gradle
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/bitrise-io/depman/pathutil"
+	"github.com/bitrise-io/go-utils/log"
+	glob "github.com/ryanuber/go-glob"
+)
+
+// Project ...
+type Project struct {
+	location string
+	monoRepo bool
+}
+
+// NewProject ...
+func NewProject(location string) (Project, error) {
+	buildGradleFound, err := pathutil.IsPathExists(filepath.Join(location, "build.gradle"))
+	if err != nil {
+		return Project{}, err
+	}
+
+	if !buildGradleFound {
+		return Project{}, fmt.Errorf("no build.gradle file found in (%s)", location)
+	}
+
+	root := filepath.Join(location, "..")
```

Comment:
if location is: `/` filepath.Join(location, "..") = `..` handle this case
also maybe filepath.Base(location) would be better

Reply:
filepath.Base returns the last element of the path.